muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/records/body/application/current-record.tsx

```diff
@@ -59,29 +59,29 @@ class CurrentRecord extends React.Component<CurrentRecordProps, CurrentRecordSta
     let workspaceEvaluation = this.props.records.current.workspace.studentAssessments.assessments.length &&
       this.props.records.current.workspace.studentAssessments.assessments[0].verbalAssessment ?
         <div dangerouslySetInnerHTML={{__html: this.props.records.current.workspace.studentAssessments.assessments[0].verbalAssessment}} 
-        className={`text rich-text text--studies-workspace-literal-assessment state-${assesmentStateClassName}`}/> : null;
+        className={`rich-text application-sub-panel__text application-sub-panel__text--course-evaluation state-${assesmentStateClassName}`}/> : null;
     
     return <div className="application-sub-panel">
-      <div className="application-sub-panel__header application-sub-panel__header--studies-detailed-info text text--studies-header" key={this.props.records.current.workspace.id}>
+      <div className="application-sub-panel__header application-sub-panel__header--studies-detailed-info" key={this.props.records.current.workspace.id}>
         {this.props.records.current.workspace.name} {this.props.records.current.workspace.nameExtension && "(" + this.props.records.current.workspace.nameExtension + ")"}
       </div>
       <div className="application-sub-panel__body application-sub-panel__body--studies-detailed-info">
+        {workspaceEvaluation}      
         <ApplicationList>
-          {workspaceEvaluation}
-          <div className="application-list__header application-list__header--studies-detailed-info text text--studies-list-header">{this.props.i18n.text.get("plugin.records.assignments.title")}</div>
+          <div className="application-list__header-container application-list__header-container--studies-detailed-info">{this.props.i18n.text.get("plugin.records.assignments.title")}</div>
```

Comment:
Why use application-list__header-container instead of application-list__header-title as used previously? Only child element appears to be text node.